tests/secure_tunneling_tests.c

```diff
@@ -48,34 +47,34 @@ struct secure_tunneling_test_context {
 static struct secure_tunneling_test_context s_test_context = {.max_threads = 1};
 
 static bool s_on_stream_start_called = false;
-static void s_on_stream_start(void *user_data) {
-    UNUSED(user_data);
-    s_on_stream_start_called = true;
-}
+// static void s_on_stream_start(void *user_data) {
```

Comment:
Super minor nitpick: Remove comments here, unless keeping it as a placeholder for later, in which case ignore this comment!